clients/s3/s3.go

```diff
@@ -146,9 +147,10 @@ func (s *Store) Merge(ctx context.Context, tableData *optimization.TableData) er
 	return os.RemoveAll(fp)
 }
 
-func LoadStore(ctx context.Context, settings *config.S3Settings) (*Store, error) {
+func LoadStore(_ context.Context, settings *config.S3Settings) (*Store, error) {
```

Comment:
Can context be removed as an arg or does it need to be here to be compatible with an interface?

Reply:
Not required!